bulk_grades/__init__.py

```diff
@@ -2,6 +2,6 @@
 Support for bulk scoring and grading.
 """
 
-__version__ = '1.0.0'
```

Comment:
this should be unnecessary; github actions do not pull anything out of pypi so these changes should take effect without a version bump